src/components/base-address/BaseAddress.vue

```diff
@@ -170,7 +170,7 @@ import {
   spaceRules
 } from '@/components/base-address/factories'
 import { AddressIF, SchemaIF } from '@bcrs-shared-components/interfaces'
-import { AddressValidationRules } from '@bcrs-shared-components/enums'
+import { AddressValidationRules } from '@bcrs-shared-components/enums/address-validation-rules'
```

Comment:
Why do you need to make this change? Looks like old code should work...?

Reply:
Gave me an error message before, now with the change it doesn't

Reply:
Weird, because AddressValidationRules is exported correctly via the index.ts file.

What's the error message?

Reply:
"has no exported member named x", which is double weird because I could import ApprovalTypes no problem when trying. SchemaIF also gets the same error message, while AddressIF does not even though they are in the same file. Might just be the editor but I am unsure.

Reply:
I see that the BaseAddress component does not import the shared enums. Try adding that (plus `lerna bootstrap --hoist`).

Reply:
Sorry, I was looking at wrong branch.

Try that Lerna command anyway,

Reply:
By the way, the old code works fine for me, so you shouldn't need to do this either.

Reply:
Old code works. I reverted this. Resolving.